enterprise/app/filter/filter.tsx

```diff
@@ -49,43 +68,88 @@ export default class FilterComponent extends React.Component<FilterProps, State>
     const startDate = (startDateParam ? moment(startDateParam) : moment().subtract(7, "days")).toDate();
     const endDate = (endDateParam ? moment(endDateParam) : moment()).toDate();
 
-    const isFiltering = Boolean(this.props.search.toString());
+    const canClearFilters = Boolean(this.props.search.toString());
```

Comment:
Will this trigger from the artifact or target filters?

Reply:
(Updated this logic so it only depends on the params controlled via the dropdown)